package.json

```diff
@@ -509,6 +509,7 @@
     "@types/madge": "^5.0.3",
     "@types/micromatch": "^4",
     "@types/mocha": "^10.0.3",
+    "@types/nanoid": "^3.0.0",
```

Comment:
This doesn't seem to be required? The version we use of `nanoid` everywhere else has types

Reply:
You're right. It was required at some point I think, but works fine without it now.